apps/meteor/app/api/server/v1/channels.ts

```diff
@@ -471,8 +465,8 @@ API.v1.addRoute(
 
 			const room = findChannelByIdOrName({
 				params: {
-					roomId: channelId,
-					roomName: channelName,
+					...(channelId && { roomId: channelId }),
+					...(channelName && { roomName: channelName }),
```

Comment:
```suggestion
					roomId: channelId,
					roomName: channelName,
```
The function will have to check each field any way. No need to do it again here.

Reply:
This endpoint is not working when only the `channelName` param is provided, so I added this change to fix it. But now I see a better approach would be to change the validation in `findChannelByIdOrName`. Tks!